permissions/ApplicationInstallation.permission

```diff
@@ -1,8 +1,8 @@
 # -*- mode: sh -*-
 
-# x-sailjail-translation-catalog =
-# x-sailjail-translation-key =
-# x-sailjail-description = Install software packages
+# x-sailjail-translation-catalog = sailjail-permissions
+# x-sailjail-translation-key = permission-la-application_installation
+# x-sailjail-description = Install applications
```

Comment:
Also this is not really applications but all kind of packages. From user point of view maybe better still talk about apps.